README.md

```diff
@@ -13,8 +13,10 @@ cd plotIt/
 
 # Initialize the git remotes
 source firstsetup.sh 
-# Load the proper environment (a la CMSSW_7_4_10)
-source setup_sl6_env.sh
```

Comment:
I fully agree with sourcing the other, see the full diff :wink: 

my current worry is that the default is compiling without cmsenv, so default execution should be without cmsenv as well... which is a bit counter intuitive as I think 'default' install for most of us is within CMSSW anyway